python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -494,11 +503,13 @@ def __set__(self, obj, val):
                 val = self._stripAttribute(obj, val, getName(self))
 
         # apply attribute filter to component items
-        if item.isA(Schema.ComponentItem):
+        if item.isA(ComponentItem):
             val = self._stripAttribute(obj, val, getName(self))
 
         self._check_type(obj, val)
-        setattr(stripProxy(obj), getName(self), val)
+        
+        GangaObject = _getGangaObject()
+        GangaObject.__setattr__(stripProxy(obj), getName(self), val)
```

Comment:
Why is the normal method lookup being subverted here? I don't think any `GangaObject` subclasses or instances should be overriding `__setattr__`.